cool-core/src/main/java/com/nus/cool/core/iceberg/aggregator/MaxAggregator.java

```diff
@@ -42,7 +42,7 @@ public void process(Map<String, BitSet> groups, FieldRS field,
             }
             if (resultMap.get(entry.getKey()).getMax() == null) {
                 BitSet bs = entry.getValue();
-                int max = 0;
+                float max = 0;
```

Comment:
InputVector::get returns int yet results in the resultMap are in float. May be we shall change them all to int?

Reply:
update the max